bevrand.uitests/cypress/integration/Profile.feature

```diff
@@ -33,7 +33,6 @@ Feature: Profile
     Then I should get a validation of success
     And I should be rerouted to the <page> page
     And My new playlist should be visible
-
```

Comment:
In other outline features there is a newline between the scenario and the examples